notebooks/Image_Analysis.ipynb

```diff
@@ -109,6 +110,58 @@
     "\n"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "7b118a48",
+   "metadata": {},
+   "source": [
+    "#### Box Plot Code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19f35cb9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "box_plot_data=[bg_vars_a, bg_vars_b]\n",
+    "\n",
+    "fig, ax = plt.subplots(dpi=100, figsize=(18,10))\n",
+    "\n",
+    "box = ax.boxplot(box_plot_data,vert=0,patch_artist=True,labels=['PSP-A','PSP-B'], \n",
```

Comment:
Labels need to come from either the dataset directly or be specified above.

Reply:
Done.